pkg/registry/authentication/tokenreview/storage.go

```diff
@@ -34,26 +34,31 @@ import (
 
 var badAuthenticatorAuds = apierrors.NewInternalError(errors.New("error validating audiences"))
 
+// REST implements a RESTStorage for tokenreview.
 type REST struct {
 	tokenAuthenticator authenticator.Request
 	apiAudiences       []string
 }
 
+// NewREST returns a RESTStorage object that will work against tokenreview.
 func NewREST(tokenAuthenticator authenticator.Request, apiAudiences []string) *REST {
 	return &REST{
 		tokenAuthenticator: tokenAuthenticator,
 		apiAudiences:       apiAudiences,
 	}
 }
 
+// NamespaceScoped fulfill rest.Scoper.
 func (r *REST) NamespaceScoped() bool {
 	return false
 }
 
+// New creates a new tokenreview object.
 func (r *REST) New() runtime.Object {
 	return &authentication.TokenReview{}
 }
 
+// Create attempts to get self subject rules in specific namespace.
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
```

Comment:
Create reviews the token (the `obj`) and records the results in tokenReview.Status.